runtime/node/package.json

```diff
@@ -16,10 +16,11 @@
   "license": "ISC",
   "dependencies": {
     "@azure/functions": "^1.2.2",
-    "botbuilder": "4.10.0-rc0",
-    "botbuilder-dialogs": "4.10.0-rc0",
-    "botbuilder-dialogs-adaptive": "4.10.0-rc0-preview",
-    "botbuilder-dialogs-declarative": "4.10.0-rc0-preview",
+    "botbuilder": "next",
```

Comment:
RC is out, could we use the 4.11.0 RC build?

Reply:
updated to 4.11.0-rc1